src/Microsoft.AspNetCore.SpaTemplates/content/Aurelia-CSharp/Properties/launchSettings.json

```diff
@@ -0,0 +1,41 @@
+{
+  "iisSettings": {
+//#if (WindowsAuth)
+    "windowsAuthentication": true,
+    "anonymousAuthentication": false,
+//#else
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+//#endif
+    "iisExpress": {
+      "applicationUrl": "http://localhost:55556/",
+//#if (RequiresHttps)
+      "sslPort": 43434
+//#else
+      "sslPort": 0
+//#endif
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+//#if (RequiresHttps)
+      "launchUrl": "https://localhost:43434/",
+//#endif
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+//#if (!WindowsAuth)
+    },
+    "Company.WebApplication1": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "http://localhost:5001/"
```

Comment:
Why is it port 5001 here, and port 5000 in some other cases?

Reply:
In 2.0.3, both the RazorPages and StarterWeb templates used 5001, while Empty and WebApi use 5000. I don't know why this decision was made. For now, I will switch the defaults it to 5000.